crates/volta-core/src/run/mod.rs

```diff
@@ -136,13 +141,20 @@ fn tool_name_from_file_name(file_name: &OsStr) -> OsString {
     }
 }
 
-fn command_for<A>(exe: &OsStr, args: A, path_var: &OsStr) -> Command
+fn command_for<A>(exe: &OsStr, args: A, path_var: &OsStr, current_dir: Option<&Path>) -> Command
```

Comment:
I don't really think this is the best way to add this option, but I wasn't sure how to do better. I attempted extracting setting this into a separate function, but then I received compiler errors about the command being moved into my new function and I was not sure how to fix it.